src/App.tsx

```diff
@@ -10,33 +11,27 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState('tab-1');
```

Comment:
id can be changed, use tabs array instead of hardcoding values
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0].id);
```